saleor/checkout/test_checkout.py

```diff
@@ -36,13 +36,13 @@ def test_checkout_clear_storage():
     checkout.storage['new'] = 1
     checkout.clear_storage()
     assert checkout.storage is None
-    assert checkout.modified == True
+    assert checkout.modified is True
```

Comment:
I know they are constants and under cpython all small ints are cached but as far as I can tell Python's specification does not guarantee anything but `None` to be a singleton. But even for readability I'd much rather just test for truthiness.